### ISSUE TITLE
PayPal Pro - Cannot redeclare class Website_Payments_Pro

### ISSUE THREAD

**Issue by Noodleyman** [cubecart/v6#1875]
during test I found that sometimes whilst making multiple purchases PayPal pro fails with the error:

`PHP Fatal error:  Cannot redeclare class Website_Payments_Pro in /homepages/26/d406520087/htdocs/modules/plugins/PayPal_Pro/website_payments_pro.class.php on line 13`

to reproduce, simply checkout once with PP Pro, then buy another item and checkout. The items also had discount codes applied, but I don't think that's part of the underlying issue.

**Comment by abrookbanks** [comment]
I've never seen this. Are you sure there is no copy of the extensions like for example:

```
/modules/plugins/PayPal_Pro/
/modules/plugins/PayPal_Pro-old/
```

**Comment by Noodleyman** [comment]
I can confirm there is only one instance of the module installed.

**Comment by abrookbanks** [comment]
Thanks I wanted to rule it out before spending an hour or more on this.

**Comment by abrookbanks** [comment]
Fixed with 1.1.13
